src/Ocis.php

```diff
@@ -306,6 +307,29 @@ private function getServiceUrlFromWebfinger(string $webfingerUrl): string
         }
         throw new InvalidResponseException('invalid webfinger response');
     }
+
+    /**
+     * saves current oCIS version in semantic versioning format ( e.g. "5.0.5" )
+     *
+     * @return string
+     * @throws InvalidResponseException
+     */
+    private function getOcisVersion(): string
```

Comment:
This function could internally save the ocis version the first time that it is called,
When it is called again, it can return the saved value, rather than always doing an API request.